test-suite-http-server-tck-netty/src/test/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+    <logger name="io.micronaut.http.client" level="TRACE"/>
```

Comment:
Might cause a lot of noise in logs

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/8499/commits/28462ad10ad23ef902662c84ebd29b87ea70c3e7